doc/2/api/errors/error-codes/api/index.md

```diff
@@ -53,5 +53,6 @@ description: Error codes definitions
 | api.process.not_enough_nodes<br/><pre>0x0202000b</pre>  | [ServiceUnavailableError](/core/2/api/errors/error-codes#serviceunavailableerror) <pre>(503)</pre> | Rejected: this cluster is disabled because there aren't enough nodes connected. | The Kuzzle cluster has not enough nodes available, and no new requests can be processed until new nodes are added |
 | api.process.unauthorized_origin<br/><pre>0x0202000c</pre>  | [UnauthorizedError](/core/2/api/errors/error-codes#unauthorizederror) <pre>(401)</pre> | The origin "%s" is not authorized. | The domain reaching out to Kuzzle is not authorized |
 | api.process.too_many_logins_requests<br/><pre>0x0202000d</pre>  | [TooManyRequestsError](/core/2/api/errors/error-codes#toomanyrequestserror) <pre>(429)</pre> | Rejected: Too many login attempts per second | The request was denied because the maximum ("limits.loginsPerSecond") number of login attempts per second has been exceeded. |
+| api.process.deprecated_embedded_sdk_action<br/><pre>0x0202000e</pre>  | [BadRequestError](/core/2/api/errors/error-codes#badrequesterror) <pre>(400)</pre> | The action %s of the controller %s has been called while it is deprecated %s. | A deprecated sdk action has been called |
```

Comment:
Technically something deprecated is something you can use even if you shouldn't because it will be removed in a future version so it's weird to have something deprecated and forbidden